app.js

```diff
@@ -76,12 +76,16 @@ Test this function by hand in the console to get it working, and when you think
 var testArray = [2,3,4]; //eslint-disable-line
 
 function sumArray(testArray){ //eslint-disable-line
-
+  var testArray = [];
+  var testSum = sum(sum(2,3)[0],4);
+  testArray[0] = testSum[0];
+  testArray[1] = '2,3,4 was passed in as an array of numbers, and 9 is their sum.';
+  return testArray;
```

Comment:
testArray shouldn't be the product of this function. It should be a new array with the sum and statement.